src/args.rs

```diff
@@ -52,77 +86,78 @@ pub struct GenerateArgs {
 
     /// Directory to create / project name; if the name isn't in kebab-case, it will be converted
     /// to kebab-case unless `--force` is given.
-    #[arg(long, short, value_parser)]
+    #[arg(long, short, value_parser, help_heading = heading::OUTPUT_PARAMETERS)]
     pub name: Option<String>,
 
-    /// Don't convert the project name to kebab-case before creating the directory.
-    /// Note that cargo generate won't overwrite an existing directory, even if `--force` is given.
-    #[arg(long, short, action)]
+    /// Don't convert the project name to kebab-case before creating the directory. Note that cargo
+    /// generate won't overwrite an existing directory, even if `--force` is given.
+    #[arg(long, short, action, help_heading = heading::OUTPUT_PARAMETERS)]
     pub force: bool,
 
     /// Enables more verbose output.
     #[arg(long, short, action)]
     pub verbose: bool,
 
-    /// Pass template values through a file
-    /// Values should be in the format `key=value`, one per line
-    #[arg(long, value_parser)]
+    /// Pass template values through a file. Values should be in the format `key=value`, one per
+    /// line
+    #[arg(long="values-file", value_parser, alias="template-values-file", value_name="FILE", help_heading = heading::OUTPUT_PARAMETERS)]
```

Comment:
Added a shorter name here to make the formatting nicer (and also to make this easier for a user to type). I left the original alias however so any existing scripts etc. can still use that name.

Reply:
thanks for keeping it as a non-breaking change.